k8s/templates/wants-gatekeeper.yaml

```diff
@@ -0,0 +1,92 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: wants-gatekeeper
+  namespace: {{ .Values.namespace }}
+data:
+  nginx.conf: |
+    worker_rlimit_nofile 8192;
+
+    events {
+      worker_connections  4096;  ## Default: 1024
+    }
+
+    http {
+      proxy_redirect          off;
+      proxy_set_header        Host            $host;
+      proxy_set_header        X-Real-IP       $remote_addr;
+      proxy_set_header        X-Forwarded-For $proxy_add_x_forwarded_for;
+      client_max_body_size    32m;
+      client_body_buffer_size 128k;
+      proxy_connect_timeout   90;
+      proxy_send_timeout      90;
+      proxy_read_timeout      90;
+      proxy_buffers           32 4k;
+      resolver                127.0.0.11; // proxy_pass with variables needs a resolver
+
+      map $http_cookie $target {
+        default "nginx-router:80/welcome.html";
```

Comment:
I'll need to come up with a simple login page for this. Otherwise this looks great!